### DISCUSSION TITLE
打不开聊天

### DISCUSSION THREAD

**Discussion by ry424515050** [BurnHair/docs#100] [分享讨论]
![9060a4829836e77ff5a622514423ada0](https://github.com/BurnHair/docs/assets/79887268/c90e77ea-4380-4c6f-b8de-383d1f6c8553)

**Comment by BennyThink** [comment]
试试直接访问

https://chat.burn.hair/

或

https://lobe.burn.hair/

**Reply by ry424515050** [reply]
还是不行

**Reply by BennyThink** [reply]
那可能需要你自己解决到 vercel 的网络问题。我近期也会想想办法，自建可以参考这里 https://github.com/orgs/BurnHair/discussions/99
